bugs/lp-1456277.txt

```diff
@@ -0,0 +1,319 @@
+=== Begin SRU Template ===
+[Impact]
+We are not allowing for ec2 mirrors to be used on non ec2 instances by default. Users can still
+configure those mirrors manually through user data, but cloud-init will not default to those mirrors
+any longer.
+
+[Test Case]
+
+```
+set -x
+# Manually deploy on a lxc container
+
+cat > setup_dev_proposed.sh <<EOF
+#/bin/bash
+add-apt-repository ppa:cloud-init-dev/proposed -y
```

Comment:
Since cloud-init is in xenial-proposed we probably generally want to use the following instead of adding the cloud-init-dev/proposed ppa (which is a different binary package that what is truly uploaded to xenial-proposed)
```
echo > setup_proposed.sh <<EOF
#/bin/bash                                                                      
                                                                                
mirror=http://archive.ubuntu.com/ubuntu                                         
echo deb $mirror $(lsb_release -sc)-proposed main | tee /etc/apt/sources.list.d/proposed.list
apt-get update -q;                                                              
apt-get install -qy cloud-init;                                                 
EOF   
```
Inst

Reply:
@blackboxsw you mentioned `xenial-proposed` but there's nothing specific to xenial, correct? Just to make sure I'm not missing part of the point.